engine/src/main/java/org/entando/entando/web/common/model/Filter.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2015-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package org.entando.entando.web.common.model;
+
+import com.agiletec.aps.system.common.FieldSearchFilter;
+import com.agiletec.aps.system.common.FieldSearchFilter.LikeOptionType;
+import org.apache.commons.lang.StringEscapeUtils;
+
+
+public class Filter {
+
+    private String attribute;
+    private String operator;
+    private String value;
+
+    public String getAttribute() {
+        return attribute;
+    }
+
+    public void setAttribute(String attribute) {
+        this.attribute = attribute;
+    }
+
+    public String getOperator() {
+        return operator;
+    }
+
+    public void setOperator(String operator) {
+        this.operator = operator;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+
+    public Filter() {
+        //
+    }
+
+    public Filter(String attribute, String value) {
+        this.attribute = attribute;
+        this.value = value;
+    }
+
+    public Filter(String attribute, String value, String operator) {
+        this(attribute, value);
+        this.operator = operator;
+    }
+
+    public String getAttributeName() {
+        return this.getAttribute();
+    }
+
+    @SuppressWarnings("rawtypes")
+    public FieldSearchFilter getFieldSearchFilter() {
+        FieldSearchFilter filter = new FieldSearchFilter(StringEscapeUtils.escapeSql(this.getAttributeName()), StringEscapeUtils.escapeSql(this.getValue()), true, LikeOptionType.COMPLETE);
+        return filter;
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
Do we need this?

Reply:
I need this in my tests to compare `RestListRequest`s